PendingReleaseNotes.md

```diff
@@ -0,0 +1,5 @@
+# v0.17 Pending Release Notes
+
+## Breaking Changes
+
+## Features
```

Comment:
This format will be necessary to allow for multiple release's worth of notes in the file.

```suggestion
# Release Notes

## v0.17 Pending Release Notes

### Breaking Changes

### Features
```